### PR TITLE
Upgrade agp version for ConfigurationCacheForAndroidIT

### PR DESCRIPTION
**Pull request by bingranl**
The new agp version contains the fix regarding build listeners which
will allow kgp run config caching test without warnings.

Fixes: n/a
Test: existing

### CODE REVIEW

**Comment by bingranl** [comment]
This CL passes ./gradlew :kotlin-gradle-plugin-integration-tests:check

**Review by gavra0 — APPROVED** [review]

**Comment by YokoCross** [comment]
It's good change

**Comment by nav-nav** [comment]
It looks good. I'm waiting for test result before merge

**Comment by nav-nav** [comment]
cherry-picked into master

**Pull request closed without merging**
